### PR TITLE
fix: redirect page type icon not properly displayed

### LINKED ISSUE #8498: redirect page type icon not properly displayed
### Astro Info

```block
Astro version            v2.10.15
Package manager          npm
Platform                 darwin
Architecture             arm64
Adapter                  astro-sst/edge
Integrations             @y_nk/cached-fetch, @astrojs/tailwind, @astrojs/react, @astrojs/sitemap
```

### If this issue only occurs in one browser, which browser is a problem?

_No response_

### Describe the Bug

redirected page build shown as lambda (even tho it's not)

image

### What's the expected result?

it should be shown as a static built page

### Link to Minimal Reproducible Example

https://stackblitz.com/edit/withastro-astro-aebx5j

### Participation

- [ ] I am willing to submit a pull request for this issue.

### PR DESCRIPTION
**Pull request by rishi-raj-jain**
fix #8498

### CODE REVIEW

**Comment by florian-lefebvre** [comment]
Shouldn't it show a different symbol for redirects?

**Comment by rishi-raj-jain** [comment]
I tried finding one but couldn't think of then read https://github.com/withastro/astro/issues/8498#:~:text=it%20should%20be%20shown%20as%20a%20static%20built%20page

File: packages/astro/src/core/build/generate.ts [hunk 1]
```diff
@@ -297,7 +297,10 @@ async function generatePage(
 		mod: pageModule,
 	};
 
-	const icon = pageData.route.type === 'page' ? green('▶') : magenta('λ');
+	const icon =
+		pageData.route.type === 'page' || pageData.route.type === 'redirect'
+			? green('▶')
+			: magenta('λ');
```
Comment (florian-lefebvre) [inline packages/astro/src/core/build/generate.ts:303]:
```suggestion
	const icon =
		pageData.route.type === 'page'
			? green('▶')
			: pageData.route.type === 'redirect'
			? cyan('🔀')
			: magenta('λ');
```

**Review by florian-lefebvre — COMMENTED** [review]
Wdyt?

File: packages/astro/src/core/build/generate.ts (hunk 1 above)
Comment (rishi-raj-jain) [reply packages/astro/src/core/build/generate.ts:303]: That more of seems like an exchange

Comment (rishi-raj-jain) [reply packages/astro/src/core/build/generate.ts:303]: Screenshot_2023-09-17-15-07-44-65_40deb401b9ffe8e1df2f1cc5ba480b12

My mind is also in sync with this right arrow proposal by chatgpt

Comment (rishi-raj-jain) [reply packages/astro/src/core/build/generate.ts:303]: Screenshot_2023-09-17-15-07-44-65_40deb401b9ffe8e1df2f1cc5ba480b12

My mind is also in sync with this right arrow proposal by chatgpt

Comment (rishi-raj-jain) [reply packages/astro/src/core/build/generate.ts:303]: @Princesseuh @sarah11918 thoughts?

Comment (florian-lefebvre) [reply packages/astro/src/core/build/generate.ts:303]: I also asked chat gpt ahah, not sure about what's the right emoji there

Comment (Princesseuh) [reply packages/astro/src/core/build/generate.ts:303]: You probably don't want to use a emoji anyway because some terminals can't render them

Comment (rishi-raj-jain) [reply packages/astro/src/core/build/generate.ts:303]: Awesome, so right now the PR looks good?

Comment (Princesseuh) [reply packages/astro/src/core/build/generate.ts:303]: Looks fine to me, we'll review it properly during the week once we're back to work

Comment (rishi-raj-jain) [reply packages/astro/src/core/build/generate.ts:303]: Awesome then, resolving this. Thanks!

File: .changeset/two-jeans-serve.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+'astro': minor
```
Comment (bluwy) [inline .changeset/two-jeans-serve.md:2]: I think this can be classified as a patch as it doesn't introduce new features.

**Review by bluwy — APPROVED** [review]

**Review by ematipico — APPROVED** [review]
Looks good to me

**Review by bluwy — APPROVED** [review]

**Pull request merged**